internal/plugin/util/pluginhelpers.go

```diff
@@ -34,6 +34,16 @@ set the ` + "`PROVIDER_AIVEN_ENABLE_BETA`" + ` environment variable to true.
 `
 }
 
+// LimitedAvailabilityDescription is a helper function that returns a description for features that are in LA.
+func LimitedAvailabilityDescription(description string) string {
```

Comment:
WDYT if we combine `BetaDescription` and `LimitedAvailabilityDescription` in a builder pattern, like [here](https://github.com/aiven/terraform-provider-aiven/blob/ee5c8524ecfb14279d48658fc39a5088bf0f534e/internal/schemautil/userconfig/desc.go)? Please let me know if this is too advanced for you to implement, I'll be willing to assist 🙂